python/lsst/pipe/supertask/activator.py

```diff
@@ -407,6 +447,15 @@ def parse_and_run(cls):
             for package in args.packages:
                 TASK_PACKAGES[package] = importlib.import_module(package)
 
+        if args.package_paths:
+            TASK_PACKAGES.clear()
+            for package_path in args.package_paths:
+                if not os.path.exists(package_path):
+                    print(package_path, ' does not exist!')
```

Comment:
There are several print outs in the code that are in the meantime for debugging and also informative. I will definitely revisit all the printouts and what to include in the logfile when this is more widely tested. 

I added a continue clause to avoid creating the fake class, in case there are more path parsed. It won't raise an exception automatically, unless no task at all are found.